doc/ngsi/orion/user_and_programmer_guide/README.md

```diff
@@ -0,0 +1,7 @@
+#User and Programmer Guide
+
+* [Introduction](./introduction.md)
+* [Creating an Orion connection](./creating_orion_connection.md)
+* [Appending data to Orion](./appending_data.md)
+* [Querying data from Orion](./querying_data.md)
+* [Issues and contact information](./issues_and_conact.md)
```

Comment:
Fixed in PR https://github.com/telefonicaid/iotp-talend-connectors/pull/4/commits/91793586c59de32feacb932ee31347e94d71284d